src/components/cc-product-card/cc-product-card.js

```diff
@@ -0,0 +1,198 @@
+      description: { type: String },
+      icon: { type: String },
+      link: { type: String },
+      keywords: { type: Array },
+      name: { type: String },
+    };
+  }
+
+  constructor () {
+    super();
+
+    /** @type {string|null} The description of the product */
+    this.description = null;
+
+    /** @type {string|null} The icon of the product */
+    this.icon = null;
+
+    /** @type {string|null} The link where the user will be redirected when the product is selected  */
+    this.link = null;
+
+    /** @type {Keyword[]} The keywords representing the product */
+    this.keywords = [];
+
+    /** @type {string|null} The name of the product */
+    this.name = null;
+  }
+
+  _onProductSelected () {
+    dispatchCustomEvent(this, 'select', this.name);
+  }
+
+  render () {
+
+    const hasKeywords = this.keywords.length > 0;
+
+    // language=HTML
+    return html`
+      <div class="${hasKeywords ? 'wrapper' : 'wrapper-no-keywords'} wrapper-base">
+        <cc-img src="${this.icon}"></cc-img>
+        <span class="name">${this.name}</span>
+        <p class="description">${this.description}</p>
+        ${hasKeywords ? html`
+          <ul class="keywords">
+            ${this.keywords.map((keyword) => {
+              return (!keyword.hidden) ? html`
+                <li>
+                  <cc-badge>${keyword.value}</cc-badge>
+                </li>
+              ` : '';
+            })}
+          </ul>
+        ` : ''}
+        <a href="${this.link}" class="select" @click="${this._onProductSelected}">${i18n('cc-product-card.select')}</a>
+      </div>
+    `;
+  }
+
+  static get styles () {
+    return [
+      // language=CSS
+      css`
+      :host {
+        display: block;
+      }
+
+      /* region RESET */
+       
+      p {
+        margin: 0;
+      }
+      
+      ul {
+        padding: 0;
+        margin: 0;
+        list-style: none;
+      }
+
+      /* endregion */
+
+      .wrapper-base {
+        display: grid;
+        padding: 1em;
+        border: 1px solid var(--cc-color-border-neutral, #aaa);
+        background-color: var(--cc-color-bg-default, #fff);
+        border-radius: var(--cc-border-radius-default, 0.25em);
+        gap: 0.75em;
+      }
+        
+      /* region wrapper */
+
+      .wrapper {
+        grid-template-areas: 
+          'icon name'
+          'keywords keywords'
+          'description description'
+          'select select';
+        grid-template-columns: min-content 1fr;
+        grid-template-rows: min-content max-content auto auto;
+      }
+
+      .wrapper-no-keywords {
+        grid-template-areas: 
+          'icon name'
+          'description description'
+          'select select';
+        grid-template-columns: min-content 1fr;
+        grid-template-rows: min-content auto auto;
+      }
+        
+      /* endregion */
+        
+      /* region grid-items */
+
+      cc-img {
+        width: 2em;
+        height: 2em;
+        border-radius: 0.2em;
```

Comment:
question: why not using `border-radius: var(--cc-border-radius-default, 0.25em);` ?